src/presenter/trip.js

```diff
@@ -0,0 +1,110 @@
+import SortView from "../view/sort.js";
+import TripDaysView from "../view/trip-days.js";
+import TripDayView from "../view/trip-day.js";
+import TripEventsView from "../view/trip-events.js";
+import TripEventView from "../view/trip-event.js";
+import EventEditView from "../view/event-edit.js";
+import EventMessageView from "../view/event-message.js";
+import {render, RenderPosition, replace} from "../utils/dom.js";
+import {groupCardsByDay} from "../utils/date.js";
+import {escEventHandler} from "../utils/dom-event.js";
+import {destinations} from "../mock/destinations.js";
+import {BlockTitle, EventMessage} from "../constants.js";
+
+export default class Trip {
+  constructor(eventsContainer, renderBlock) {
```

Comment:
https://github.com/htmlacademy-ecmascript/1033161-big-trip-12/blob/0f00e6621d2cfc2580f8bdbff03043abb20d0886/src/presenter/trip.js#L23-L24

```suggestion
  constructor(eventsContainer, renderBlock) {
    this._tripCards = [];
```